tests/unit/views/pacbio/PacbioPoolCreate.spec.js

```diff
@@ -77,7 +79,7 @@ describe('PacbioPoolCreate', () => {
   })
 
   const wells = dataToObjectById({
-    data: Data.PacbioPlatesRequest.data.included.slice(0, 4),
+    data: pacbioPlatesRequestFactory.content.included.slice(0, 4),
```

Comment:
If you move this to the storeData you can reuse what you had in the tests/unit/components/pacbio/PacbioLabwareSelectedList.spec.js